token-metadata/0xB6eD7644C69416d67B522e20bC294A9a9B405B31/metadata.json

```diff
@@ -3,5 +3,20 @@
     "symbol": "0XBTC",
     "address": "0xB6eD7644C69416d67B522e20bC294A9a9B405B31",
```

Comment:
0x7CB5454213b5ffF7e60ad63768d77ea46E621b3A

Reply:
could you help me understand what this address is?

Reply:
IS https://etherscan.io/token/0xb6ed7644c69416d67b522e20bc294a9a9b405b31 which is the contract for 0xBitcoin Token. 
 Tthe file im editing 0xb6ed7644c69416d67b522e20bc294a9a9b405b31.metadata ???